python/Ice/coroutine/client.py

```diff
@@ -5,40 +5,70 @@
 
 import signal
 import sys
+import time
 import Ice
 
-Ice.loadSlice('Fetcher.ice')
+Ice.loadSlice('Fibonacci.ice')
 import Demo
 
 
 def run(communicator):
+    fibonacci = Demo.FibonacciPrx.uncheckedCast(communicator.propertyToProxy('Fibonacci.Proxy'))
 
-    fetcher = Demo.FetcherPrx.checkedCast(communicator.propertyToProxy('Fetcher.Proxy'))
-    if not fetcher:
-        print("invalid proxy")
-        sys.exit(1)
+    try:
+        v = fibonacci.compute(-1)
+        print(f"fibonacci(-1) = {v:!r}")  # unexpected
```

Comment:
I'd add or replace the print with `assert False`

Reply:
Fixed.